### PR TITLE
description paragraph added

### PR DESCRIPTION
**Pull request by barakadanny**
added a paragraph to describe the website

### CODE REVIEW

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @barakadanny , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Implement the requirements

## Required Changes ♻️

- [x] please make sure to use the right linters files and delete the others and also make sure that they are passing 

- [x] You should add a Descriptive branch name, title, and short summary that tells in detail what you've done in Pull Request
- [x]  It would be better if you add the test.md file to `.gitignore` file as required

- [x] Please make sure next time to use the[ rules ](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)of writing Desctiprive commit messages in order for your PR will be pro 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -9,9 +9,8 @@
 </head>
 <body>
     <div class="main">
-        <h1>This is just for testing</h1>
+        <h1 class="header">This is just for testing</h1>
     </div>
```
Comment (1GORDON) [inline index.html:13]: Hello @barakadanny. 
- [x] [Optional] You may try to stick to the text provided in the project requirements. "Hello Microverse!"

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+# Ignore test file
+test.md
```
Comment (1GORDON) [inline .gitignore:2]: Great job so far @barakadanny.

- [x] [Optional] The .gitignore file should include both the node_modules and the test.md file.

**Review by 1GORDON — APPROVED** [review]
Hi @barakadanny, 

- You have done a great job implementing the required changes. ✅.
- ### **Project Approved ✅**

Approved


Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉 


 ## Optional Changes
_Please check the inline comments below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 2 above)
Comment (barakadanny) [reply .gitignore:2]: I will take care of that.

File: index.html (hunk 1 above)
Comment (barakadanny) [reply index.html:13]: alright.

**Pull request merged**